baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -607,6 +607,25 @@ export class EchoClient {
     this.initialize();
     return this.innerApiCalls.wait(request, options, callback);
   }
+/**
+ * Check the status of the long running operation returned by the wait() method.
+ * @param {String} name
+ *   The operation name that will be passed.
+ * @returns {Promise} - The promise which resolves to an object.
+ *   The decoded operation object has result and metadata field to get information from.
+ *
+ * @code sample:
+ *   const decodedOperation = await checkWaitProgress(name);
+ *   console.log(decodedOperation.result);
+ *   console.log(decodedOperation.metadata);
+ *
+ */
+  async checkWaitProgress(name: string): Promise<LROperation<protos.google.showcase.v1beta1.WaitResponse, protos.google.showcase.v1beta1.WaitMetadata>>{
+    const request = new operationsProtos.google.longrunning.GetOperationRequest({name});
+    const [operation] = await this.operationsClient.getOperation(request);
+    const decodeOperation = new gax.Operation(operation as operationsProtos.google.longrunning.Operation, this.descriptors.longrunning.wait, gax.createDefaultBackoffSettings());
```

Comment:
Why the cast?

Reply:
Oh, yep, this cast should not be here - `this.operationsClient.getOperation()` should already return an `Operation`.

Reply:
so currently this `getOperation` returns `CancellablePromise<ResultTuple>`, because we wanna this promise can be cancelled.
 And the `ResultTuple` is type `[ResponseType | [ResponseType], NextPageRequestType | undefined, RawResponseType | undefined]`.

Reply:
We has an offline discussion and decided to fix this type from gax. This PR will be hold until the change is made there. Thank you!